lib/membrane_live_audio_mixer.ex

```diff
@@ -208,6 +239,24 @@ defmodule Membrane.LiveAudioMixer do
       else: {[], state}
   end
 
+  @impl true
+  def handle_parent_notification({:start_mixing, _latency}, _context, %{started?: true} = state),
+    do: {[], state}
+
+  @impl true
+  def handle_parent_notification(
+        {:start_mixing, _latency},
+        _context,
+        %{stream_format: nil} = state
+      ) do
+    Membrane.Logger.warn("Can't start mixing when `stream format` is nil")
```

Comment:
Aren't we already raising if the `stream_format==nil` and the the mixer is in "manual" mode?

Reply:
In the function pattern match, there is no check for `latency` == nil, so there is a situation in which it will be called.
If you pass latency you can still send `start_mixing` notification. I don't have a use case for it in Membrane Live so I can remove this option if you think so.

Reply:
let's leave it the way it is ;)